### PR TITLE
Fix debug assertion when exporting invalid bpm as file tag

### PR DESCRIPTION
**Pull request by uklotzde**

### CODE REVIEW

File: src/track/taglib/trackmetadata_common.h [hunk 1]
```diff
@@ -71,8 +71,11 @@ inline TagLib::String uuidToTString(
 
 inline QString formatBpm(
         const TrackMetadata& trackMetadata) {
-    return Bpm::valueToString(
-            trackMetadata.getTrackInfo().getBpm().value());
+    const Bpm bpm = trackMetadata.getTrackInfo().getBpm();
+    if (!bpm.isValid()) {
+        return {};
+    }
+    return Bpm::valueToString(bpm.value());
```
Comment (Holzhaus) [inline src/track/taglib/trackmetadata_common.h:78]: Should we make `valueToString` a member of the Bpm class (`bpm.toString()`) and make the check internally?

**Review by Holzhaus — APPROVED** [review]
Sorry for the noise :D At least we notice if we didn't handle this case now.

File: src/track/taglib/trackmetadata_common.h (hunk 1 above)
Comment (uklotzde) [reply src/track/taglib/trackmetadata_common.h:78]: Probably, because now the check is done twice.

Comment (uklotzde) [reply src/track/taglib/trackmetadata_common.h:78]: Let's do this separately. I didn't check what else is affected and if it applies to other values.

**Pull request merged**